app/components/Play/Play.js

```diff
@@ -1,32 +1,37 @@
 import React, { Component } from 'react';
-import { PickEmotion } from '../PickEmotion/PickEmotion';
+import PickEmotion from '../PickEmotion/PickEmotion';
 import ImageCapture from '../ImageCapture/ImageCapture';
 import { EmotionResults } from '../EmotionResults/EmotionResults';
+import { analyzeResponse } from '../../helper.js';
 
 export default class Play extends Component {
   constructor() {
     super()
     this.state = {
-      emotions: {}
+      emotions: []
     }
   }
 
-  analyzeEmotions(faceURL) {
+  analyzeEmotions(faceBlob) {
+    const body = faceBlob;
+
+    const headers = new Headers({
+      'Content-Type': 'application/octet-stream',
+      'Ocp-Apim-Subscription-Key': '5044b8a6914442cfbb74003672a494a0'
+    })
+
+
     fetch('https://westus.api.cognitive.microsoft.com/emotion/v1.0/recognize', {
         method: 'POST',
-        header: {
-            'Content-Type': 'application/json',
-            'Ocp-Apim-Subscription-Key': '3119bdf7116e4afebc7b8d910aa343d9'
-        },
-        body: `{'url': '${faceURL}'}`
+        headers: headers,
+        body: faceBlob
     })
     .then((response) => {
-      console.log('success');
       response.json();
```

Comment:
Yes, thank you - this was originally response => response.json(). I'll change it back.

It got edited a few times when I was trying to pinpoint what exactly I was getting from the API call. Turns out I'm just getting the preflight response, which doesn't have a body - just headers.